contracts/multisig/src/msg.rs

```diff
@@ -73,6 +73,19 @@ pub enum ExecuteMsg {
     /// Resumes routing after an emergency shutdown
     #[permission(Elevated)]
     EnableSigning,
+
+    /// Update signing parameters. Callable only by governance.
+    /// Each parameter is optional - `None` values keep the current configuration unchanged.
+    /// This allows updating parameters individually or in combination.
+    /// Updates only apply to future signing sessions, not currently active ones.
```

Comment:
this is true for block expiry, but not other configs though. For example if we ever allow to change the `rewards_contract` it would be updated for future signature submissions even for current active sessions

Reply:
In the future we can add a dedicated Params object within the config to distinguish these things perhaps